Ds-Session-4/employee/NameComparator.java

```diff
@@ -0,0 +1,20 @@
+package employee;
+
+import java.util.Comparator;
+
+/**
+ * used for imposing total ordering on Employee
+ * @author Ishita_Mathur
+ *
+ */
+public class NameComparator implements Comparator<Employee>
+{
```

Comment:
What is the use of `Comparator<Employee>` interface in this class? Please explain.